README.md

```diff
@@ -5,11 +5,33 @@ See https://www.nanorfe.com for more info.
 Below is information for building the firmware on Linux.
 
 ## Installing the compiler
-On any recent Debian based installation:
+On any recent Ubuntu based installation:
 ``` 
 apt install arm-gcc-none-eabi python3-serial
```

Comment:
You are almost right. I made a 'typo' for Debian and Ubuntu it is:
-  gcc-arm-none-eabi

Sorry about that.